packages/features/ee/organizations/lib/service/onboarding/OrganizationOnboardingSetup.md

```
@@ -0,0 +1,170 @@
+The `OrganizationOnboardingFactory` class determines which onboarding service to use:
+
+```typescript
+OrganizationOnboardingFactory.create(user)
+  → BillingEnabledOrgOnboardingService | SelfHostedOnboardingService
+```
+
+### Decision Logic
+
+The factory uses the following logic to determine which service to instantiate:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)